docs/pages/x/api/date-pickers/date-calendar.json

```
@@ -199,6 +199,18 @@
       "description": "Custom component for day.\nCheck the [PickersDay](https://mui.com/x/api/date-pickers/pickers-day/) component.",
       "default": "PickersDay",
       "class": null
+    },
+    {
+      "name": "monthButton",
+      "description": "Button displayed to render a single month in the \"month\" view .",
+      "default": "MonthCalendarButton",
+      "class": null
+    },
+    {
+      "name": "yearButton",
+      "description": "Button displayed to render a single month in the \"month\" view .",
```

I think this should be:
```suggestion
      "description": "Button displayed to render a single year in the \"year\" view .",
```